app/services/search/algolia_search.rb

```diff
@@ -0,0 +1,50 @@
+class Search::AlgoliaSearch
```

Comment:
I foresee some confusion in my future about whether I need to look in search/algolia_search.rb or search/search_builder.rb. But I can't think of an excellent alternative. I wonder about calling it AlgoliaCaller or AlgoliaDispatcher or AlgoliaSearchRequest.